src/styles/blocks/starReviewsPrice.scss

```diff
@@ -0,0 +1,77 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  width: 166px;
+  margin-block: 8px;
+  gap: 17px;
+
+  &-amount {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-end;
+    height: 16px;
+    width: 53px;
+    margin: 6px 0 10px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+    text-align: right;
+    color: $main-text-color;
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  margin: 8px 0 12px;
+  gap: 4px;
+
+  &__star {
+    display: flex;
+    flex-direction: row;
+    background-image: url(/src/images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
The path to the image should be relative to the CSS file. If the image is not displaying correctly, check that the path is correct based on the project structure.